.eslintignore

```diff
@@ -1,5 +1,6 @@
-# FIXME: Remove after switching to playwright
-e2e/browser
+# e2e browser testApp has it's own separate eslint configuration:
+e2e/browser/testApp
```

Comment:
@NSeydoux we should ideally find a way to not have a separate eslint configuration for the testApp, or maybe use a `post-install` script to install the dependencies for the testApp such that linting won't fail at that boundary.

Reply:
The `post-install` script seems like a good idea. I'll see how it looks like locally.